src/lando/jinja.py

```diff
@@ -203,23 +203,23 @@ def linkify_revision_ids(text: str) -> str:
     return re.sub(search, replace, str(text), flags=re.IGNORECASE)
 
 
-def linkify_transplant_details(text: str, transplant: dict) -> str:
+def linkify_transplant_details(text: str, landing_job: LandingJob) -> str:
```

Comment:
Could we make this a method on the `LandingJob` instead of using a template filter? ie calling this would be `LandingJob.linkify_transplant_details(text)`.

Reply:
I don't think this functionality belongs on the model, specifically adding the html tags, etc... it's a specific UI feature that probably still belongs in a jinja filter, though we should probably rework these legacy filters at some point. Though I think the actual URL value should be stored on the model (see my comment on Slack earlier).

There are also two other similar methods in this module that take in a LandingJob and do something similar with the status.